static-site/content/featured-analyses.yaml

```diff
@@ -15,10 +15,15 @@
   url: /groups/inrb-mpox/clade-I
 
 - name: H5N1 cattle outbreak
-  description: Analysis of the ongoing influenza A/H5N1 cattle outbreak
+  description: Analysis of the ongoing B3.13 outbreak
   img: h5n1_cattle_outbreak_tree.png
   url: /avian-flu/h5n1-cattle-outbreak/genome
 
+- name: H5N1 D1.1 outbreaks
```

Comment:
Suggestion: add "avian" somewhere in the title (general public doesn't know that H5N1 implies avian)

Reply:
Renamed to "Avian influenza (D1.1)".